doc/reference/reference_lua/msgpack.rst

```diff
@@ -234,34 +208,172 @@ with the MsgPack format name and encoding on the right.
 
 .. function:: cfg(table)
 
-    Some MsgPack configuration settings can be changed, in the
-    same way that they can be changed for json.
-    See :ref:`Module JSON <json-module_cfg>` for a list of some configuration settings.
-    (The same configuration settings exist for json, for MsgPack, and for  :ref:`YAML <yaml-module>`.)
+    Some MsgPack configuration settings can be changed.
+
+    The values are all either integers or boolean ``true``/``false``.
+
+    .. container:: table
+
+        .. rst-class:: left-align-column-1
+        .. rst-class:: center-align-column-2
+        .. rst-class:: left-align-column-3
+
+        +---------------------------------+---------+-----------------------------------------------+
+        | Option                          | Default | Use                                           |
+        +=================================+=========+===============================================+
+        | ``cfg.encode_max_depth``        |   128   | Max recursion depth for encoding              |
+        +---------------------------------+---------+-----------------------------------------------+
+        | ``cfg.encode_deep_as_nil``      |  false  | A flag saying whether to crop tables           |
+        |                                 |         |  with nesting level deeper than               | 
+        |                                 |         | ``max_depth``.                                |
```

Comment:
To avoid confusion, please specify the full name:

max_depth --> ``cfg.encode_max_depth`